cypress/integration/orders/orders.PoSearch.spec.js

```diff
@@ -0,0 +1,51 @@
+import NewOrder from '../../support/fragments/orders/newOrder';
+import BasicOrderLine from '../../support/fragments/orders/basicOrderLine';
+import TestType from '../../support/dictionary/testTypes';
+import Orders from '../../support/fragments/orders/orders';
+import TopMenu from '../../support/fragments/topMenu';
+import DateTools from '../../support/utils/dateTools';
+import SearchHelper from '../../support/fragments/finance/financeHelper';
+
+describe('orders: Test PO search', () => {
+  const order = { ...NewOrder.defaultOrder };
+  const orderLine = { ...BasicOrderLine.defaultOrderLine };
+  const locationName = 'Main Library';
```

Comment:
there are several places with Main Library. Let's try to move it into one place
![image](https://user-images.githubusercontent.com/93538670/158143909-96117869-edf1-4825-b2fb-a71572caa140.png)

Reply:
Updated